be/routes/api.php

```diff
@@ -21,15 +21,11 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
-});
-
 // Authentication
 Route::post('register', [AuthController::class, 'register']);
 Route::post('login', [AuthController::class, 'login']);
 
-// Protected Routed
+// Protected Routes
 Route::middleware(['auth:sanctum'])->group(function () {
     Route::post('logout', [AuthController::class, 'logout']);
     Route::resource('lessons', LessonController::class);
```

Comment:
- is this the only changes? I'm expecting you remove unnecessary routes because you use `resource`.